doc/changes/names.inc

```diff
@@ -426,6 +426,8 @@
 
 .. _Senwen Deng: https://snwn.de
 
+.. _Alessandro Tonin: https://wysscenter.ch/team/alessandro-tonin
```

Comment:
@Lychfindel I used the `:newcontrib:` above and realized we needed an entry in `names.inc`, feel free to push to change the URL if this is not the one you want!

Reply:
Sure thanks a lot, then I will remove it from my other PR!